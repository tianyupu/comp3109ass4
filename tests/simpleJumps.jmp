  goto L1;
L1:
  return 1;
