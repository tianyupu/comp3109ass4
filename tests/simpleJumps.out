L1:
  return 1;
